### ISSUE TITLE
Clustering: 'df.sort' is depreciated 

### ISSUE THREAD

**Issue by epratheeban** [pyannote/pyannote-video#23]
Hi Herve,

I'm using Ananaconda3 version 4.2.0. When I try to use clustering, It shows the below error and I learnt that from [here](https://stackoverflow.com/questions/19332171/difference-between-sort-values-and-sort-index)  `df.sort`  is depreciated in pandas version >= 0.17.0. 

It is also to be noted that moviepy throws an [error](https://github.com/Zulko/moviepy/issues/376)  and solved it by downgrading numpy from 1.12.1 to 1.11.0. 

```
AttributeError                            Traceback (most recent call last)
<ipython-input-12-85536210c8be> in <module>()
----> 1 face_tracks, embeddings = clustering.model.preprocess("/home/cvig/pyannote-data/oviya.embedding.txt")
      2 face_tracks.get_timeline()

/home/cvig/anaconda3/envs/pyannote/lib/python3.5/site-packages/pyannote/video/face/clustering.py in preprocess(self, embedding)
     73         data = read_table(embedding, delim_whitespace=True,
     74                           header=None, names=names)
---> 75         data.sort(columns=['track', 'time'], inplace=True)
     76         starting_point = Annotation(modality='face')
     77         for track, segment in data.groupby('track').apply(self._to_segment).iteritems():

/home/cvig/anaconda3/envs/pyannote/lib/python3.5/site-packages/pandas/core/generic.py in __getattr__(self, name)
   2968             if name in self._info_axis:
   2969                 return self[name]
-> 2970             return object.__getattribute__(self, name)
   2971 
   2972     def __setattr__(self, name, value):

AttributeError: 'DataFrame' object has no attribute 'sort'
```
So I tried to downgrade the pandas version to 0.16.2, but it shows this below error

```
(pyannote) cvig@cVIG-comp:~$ conda install pandas=0.16.2
Fetching package metadata .......
Solving package specifications: ....
UnsatisfiableError: The following specifications were found to be in conflict:
  - opencv -> hdf5 1.8.17|>=1.8.17
  - opencv -> libpng 1.2.50|1.5.13
  - opencv -> numpy 1.11*|1.12*|1.5*|1.6*|1.7*|1.8*|1.9*
  - opencv -> python 2.6*|2.7*|3.4*|3.6*
  - pandas 0.16.2*
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by hbredin** [comment]
Thanks for reporting this, Pratheeban.

However, it looks like there are two different issues, here:
- I'd gladly consider a pull request solving the first one (i.e. deprecated `DataFrame.sort`).
- As for the second one, I guess this is an issue that should be solved in `moviepy`.

**Comment by hbredin** [comment]
Looks like the moviepy issue has been solved upstream.

The culprit is probably this hard requirement in setup.py: https://github.com/pyannote/pyannote-video/blob/master/setup.py#L50

I don't have time to work on this right now but I'd be happy to merge a pull request updating this requirement as well (with some kind of proof this solves the issue).

**Comment by epratheeban** [comment]
I will work on it and report  with PR very soon!  As of now I used a dirty trick by using two different environment.

**Comment by hbredin** [comment]
Fixed by #27
